### PR TITLE
dnsdist: Add ChangeLog and secpoll update for DNSdist 1.9.0-alpha3

### PR DESCRIPTION
**Pull request by rgacogne**
### Short description

Remaining items:
- [ ] set the release date
- [x] merge #13280 

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: docs/secpoll.zone [hunk 1]
```diff
@@ -516,4 +516,5 @@ dnsdist-1.8.0-rc2.security-status                          60 IN TXT "2 Unsuppor
 dnsdist-1.8.0-rc3.security-status                          60 IN TXT "2 Unsupported pre-release (no known vulnerabilities)"
 dnsdist-1.8.0.security-status                              60 IN TXT "1 OK"
 dnsdist-1.8.1.security-status                              60 IN TXT "1 OK"
-dnsdist-1.9.0-alpha1.security-status                       60 IN TXT "1 Unsupported pre-release (no known vulnerabilities)"
+dnsdist-1.9.0-alpha1.security-status                       60 IN TXT "2 Unsupported pre-release (no known vulnerabilities)"
```
Comment (Habbie) [inline docs/secpoll.zone:519]: I know conflict resolution will remind you as well, but alpha1 of course has the http/2 issue

File: pdns/dnsdistdist/docs/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,73 @@
...
+
+  Please review the :doc:`Upgrade Guide <../upgrade_guide>` before upgrading.
+
+  .. change::
+    :tags: New Features, Protobuf
+    :pullreq: 13185
+
+    Log Extended DNS Errors (EDE) to protobuf
+
+    .. change::
```
Comment (Habbie) [inline pdns/dnsdistdist/docs/changelog.rst:16]: remove two spaces

File: pdns/dnsdistdist/docs/conf.py [hunk 3]
```diff
@@ -92,7 +92,7 @@
 changelog_render_changeset = "https://github.com/PowerDNS/pdns/commit/%s"
 
 changelog_sections = ['New Features', 'Improvements', 'Bug Fixes', 'Removals']
-changelog_inner_tag_sort = ['Security', 'DNS over HTTPS', 'DNS over TLS', 'DNSCrypt', 'DNSTAP', 'Protobuf', 'Performance', 'Webserver', 'Metrics']
+changelog_inner_tag_sort = ['Security', 'DNS over QUIC', 'DNS over HTTPS', 'DNS over TLS', 'DNSCrypt', 'DNSTAP', 'Protobuf', 'Performance', 'Webserver', 'Metrics']
```
Comment (Habbie) [inline pdns/dnsdistdist/docs/conf.py:95]: not a comment but a question, assuming you know: what does this do? :sweat_smile:

**Review by Habbie — CHANGES_REQUESTED** [review]
approved after the two spaces are gone. I did not check that the changelog is complete (but I'm happy to do that around tagging time)

File: pdns/dnsdistdist/docs/conf.py (hunk 3 above)
Comment (rgacogne) [reply pdns/dnsdistdist/docs/conf.py:95]: :D It is used by the doc generation process to group entries with identical tags together in the ChangeLog. This is not very useful, to be honest :)

File: docs/secpoll.zone (hunk 1 above)
Comment (rgacogne) [reply docs/secpoll.zone]: Fixed, thanks!

File: pdns/dnsdistdist/docs/changelog.rst (hunk 2 above)
Comment (rgacogne) [reply pdns/dnsdistdist/docs/changelog.rst]: Done!

**Review by Habbie — APPROVED** [review]

**Comment by rgacogne** [comment]
Updated for 1.9.0-alpha3.

**Review by Habbie — APPROVED** [review]

**Pull request merged**